lib/Applify.pm

```diff
@@ -293,11 +291,54 @@ sub _exit {
   exit $reason;
 }
 
+sub _exit_invalid_argv {
+  my ($self, $err) = @_;
+  $err =~ s!\sat\s.*!!s unless $ENV{APPLIFY_VERBOSE};
+
+  for my $option (@{$self->options}) {
+    my ($name, $arg) = @$option{qw(name arg)};
+    last if $err =~ s!"$name"!--$arg!;
+  }
+
+  $self->print_help;
+
+  local $! = 1;
```

Comment:
Good to autoflush.

Reply:
It’s not autoflush, it sets the exit code to 1 instead of 255.